packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -426,14 +425,48 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
     type: SettingsType.MAP,
     valueDefinition: {
       description: ``,
-      type: SettingsType.ANY,
+      type: SettingsType.SHAPE,
+      properties: {
+        dependencies: {
+          description: ``,
```

Comment:
Probably should make a short description; useful should someone run `yarn config get packagesExtensions[...]`